pkg/storaged/lvol-tabs.jsx

```diff
@@ -396,6 +529,117 @@
     });
 }
 
+const StructureDescription = ({ client, lvol }) => {
+    const struct = lvol.Structure;
+
+    if (!struct)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230912-094641-c7f396be-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)